### PR TITLE
Finance: Thousands Separators

### PR DESCRIPTION
**Pull request by ECWireless**
Fixing issue #1065

### CODE REVIEW

File: apps/finance/app/src/lib/utils.js [hunk 1]
```diff
@@ -28,3 +28,8 @@ export function formatTokenAmount(
     formatDecimals(round(amount / Math.pow(10, decimals), rounding), 18)
   )
 }
+
+// Thousands Separator
+export function separateThousands(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```
Comment (sohkai) [inline apps/finance/app/src/lib/utils.js]: This doesn't work if the string has a decimal, e.g. (`'1.23456'`).

We should do the replace on only the whole number ahead of the decimal.

Comment (ECWireless) [reply apps/finance/app/src/lib/utils.js]: Okay, I just made a new commit that should fix this.

**Comment by sohkai** [comment]
Actually @ECWireless, @bpierre and I were discussing this, and realized from [Ethers that the regex may not work well everywhere](https://github.com/ethers-io/ethers.js/blob/061b0eae1d4c570aedd9bee1971afa43fcdae1a6/src.ts/utils/units.ts#L69-L107).

We started talking about including some formatting utilities in aragonUI instead to do this: https://github.com/aragon/aragon-ui/issues/719

**Comment by Evalir** [comment]
Considering we will be adding formatting utilities in aragonUI, should we close this? @sohkai @bpierre

**Comment by sohkai** [comment]
Let's keep it here and update it when a new aragonUI version is released with the utility.

**Comment by ECWireless** [comment]
@sohkai Is there a timeline for when the new AragonUI version is supposed to be released? It looks like a new one is released roughly every 2 months, but figured I'd ask...

**Comment by sohkai** [comment]
@ECWireless 

> Is there a timeline for when the new AragonUI version is supposed to be released? It looks like a new one is released roughly every 2 months, but figured I'd ask...

Haha yes... we have not been very good at making incremental releases lately but we've included a bunch of updates to aragonUI and will be releasing soon.

@bpierre and I were discussing the formatting API on Friday, so we might wait just a bit longer on this—let's see.

**Comment by ECWireless** [comment]
Closing in favor of #1145, since this PR uses a forked repo that has been deleted.

**Pull request closed without merging**